### ISSUE TITLE
Standing Desk Frame with Precise Height Adjustability/Controls

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/standing-desk#31]
Came across this frame on Amazon this evening while searching for something unrelated:

UK store has more options: https://www.amazon.co.uk/-/pt/dp/B07HFZP1Q3/?th=1
![image](https://user-images.githubusercontent.com/194400/150699630-c3ec6ab0-b590-4267-9067-0c01541afbf6.png)

Available from Spain too: https://www.amazon.es/-/pt/dp/B07HFZLYNH/?th=1
![image](https://user-images.githubusercontent.com/194400/150699606-115f95f4-b776-47c3-ae4e-f83f6c69d08b.png)

Requires further research, but this is a compelling price point! 💭

**Comment by nelsonic** [comment]
@hyprstack 👋   _very_ curious to know what you decide regarding your standing desk frame. 💭

**Comment by hyprstack** [comment]
@nelsonic decided to go with this frame and desktop

<img width="1405" alt="Screenshot 2022-05-17 at 22 54 15" src="https://user-images.githubusercontent.com/5845311/168916605-b2e0c1ed-ac0f-4e8e-8aed-9bf374a73e9c.png">

**Comment by nelsonic** [comment]
@hyprstack 140cm width is great! if you have the space for more depth go for the 80cm. You won't regret it.
That combo of white legs and maple (or bamboo) top is super slick. 
Look forward to seeing a photo once you've got it all set up. 👍

**Comment by hyprstack** [comment]
@nelsonic Actually hadn't realised the extra depth option there. Will definitely get that option 😄

**Comment by hyprstack** [comment]
@nelsonic oh no, it shortens the length though. Bummer!

**Comment by nelsonic** [comment]
Yeah if it's between having more depth and less width, probably opt for the more width to accommodate the bigger monitor and a desk light. 💭

**Comment by nelsonic** [comment]
Considering: https://www.amazon.es/-/pt/dp/B07HFXLNJ7?th=1
<img width="1574" alt="image" src="https://user-images.githubusercontent.com/194400/206026659-03e34d1e-24f2-4a1c-b3fd-49c4908535f1.png">

With the Bamboo table top:
https://www.ikea.com/pt/en/p/anfallare-table-top-bamboo-00465141/
![image](https://user-images.githubusercontent.com/194400/205420559-92d6d040-a1ca-406d-8da1-7e3a2406b93a.png)

**Comment by hyprstack** [comment]
I'd probably get a model up and go for the E5 at least (dual motor and max weight of 100Kg). The E1 is single motor which means only one leg will be doing the pushing. I've also helped Becca assemble her E5 frame and the noise it makes compared to mine (E8 frame) is less "solid" if you know what I mean.

![Screenshot 2022-12-07 at 08 57 14](https://user-images.githubusercontent.com/5845311/206133825-cf1767eb-98d0-4b7a-8088-b6d3d3e1d5d1.png)

Depending on how fussy you are about the keypad, the touch and feel of the digital keypad is also significantly different (touch screen Vs analog buttons) and the digital keypad comes with child-lock functionality. With the E1 frame it has two options for keypads.

![Screenshot 2022-12-07 at 08 59 27](https://user-images.githubusercontent.com/5845311/206134381-03eec4d8-bd68-4feb-a3a3-d151fd413296.png)

![Screenshot 2022-12-07 at 08 57 46](https://user-images.githubusercontent.com/5845311/206133938-42e2cd3e-f4b5-4087-ab1b-88aa70195cb3.png)

![Screenshot 2022-12-07 at 08 58 06](https://user-images.githubusercontent.com/5845311/206134031-efc08ee7-fced-4ce8-8b99-0da356c5ec6b.png)

![Screenshot 2022-12-07 at 08 58 20](https://user-images.githubusercontent.com/5845311/206134085-810223ea-e98b-42f0-ab9b-91632ed03699.png)

Good call on the table top though. Just means you'll need to drill your own holes. But that's not an issue if you got the tools :)

**Comment by nelsonic** [comment]
Solid insights @hyprstack 👍 
Glad to know the `E8` frame is working well for you.
The `E8` is quite a lot more expensive ... https://www.amazon.es/dp/B09S8H415P
![image](https://user-images.githubusercontent.com/194400/206246383-d917b91d-bfbf-4b25-9007-5da35bd4dc85.png)

But I'm guessing it's a case of "you get what you pay for" ... 🤑 
The **`Child Lock`** feature is one I hadn't considered before but now I'm thinking is pretty much _essential_ ... 💭 
Thanks for up-selling me to a frame that's _double_ the price! 💸 😜

**Comment by nelsonic** [comment]
_Definitely_ more than I want to spend on a desk ... 💸 
https://www.flexispot.es/esben-escritorio-con-altura-regulable-standing-desk.html
<img width="1595" alt="image" src="https://user-images.githubusercontent.com/194400/207456242-39cb7c72-7490-4421-a48c-c84e573946c1.png">

But it's hella stylish
![image](https://user-images.githubusercontent.com/194400/207456887-0ed7a75c-88f9-4286-ab9d-ec4a858d129f.png)

https://www.amazon.co.uk/FLEXISPOT-Electric-Standing-Adjustable-Chargers/dp/B09KN4YSQ2?th=1

# 💸 🙄
